avm/src/daemon/ctrl.rs

```diff
@@ -996,6 +1412,385 @@ impl ControlPort {
     }
   }
 
+  pub async fn dsrrun(
```

Comment:
all of these top-level handlers except `dsmonly` and `dswonly` can also be collapsed, while those 2 can be collapsed with each other. likewise, these `_inner` handlers can be collapsed

Reply:
I didn't care for the duplication, but I couldn't see how to collapse them, since they each differ by calling a different inner function. And the inner functions all differ on the exact details of how they set up the handler memory that's serialized and which endpoint they call, so they can't really be collapsed.